graph/auxiliary/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ost-anchor",
+  "license": "UNLICENSED",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build",
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ openst/ost-anchor",
```

Comment:
Is this going to work without auth token? Also, do we need it or we can remove for now?

Reply:
Removed